safari/.gitignore

```diff
@@ -0,0 +1,7 @@
+xcuserdata/
+*.xcworkspace
+build/
+fastlane/report.xml
+fastlane/Preview.html
+fastlane/screenshots/**/*.png
+fastlane/test_output
```

Comment:
These fastlane ignores were added by Sindre. Supposedly for the auto deployment in the future